app/auth/views/mfa.py

```diff
@@ -33,27 +44,49 @@ def mfa():
     otp_token_form = OtpTokenForm()
     next_url = request.args.get("next")
 
+    if request.cookies.get("mfa"):
```

Comment:
I wonder if we can use `session` here instead of using cookies directly?

Reply:
The session contains server session variables. Cookies are associated with the request object.

Reply:
I'm not sure to understand, can you elaborate please? Why can't we use `session.get()` here?

Reply:
The client sends the 'mfa' cookie to the server. To access this value, we need the request object.  Session is used to store MFA_USER_ID for example.